host-src/tool/dc-tool.c

```diff
@@ -279,7 +281,11 @@ char serial_getc()
     retval = serial_read(&tmp, 1);
     if (retval == -1) {
         printf("serial_getc: read error!\n");
-	tmp = (char)NULL;
+#ifndef __MINGW32__		
+		tmp = (char)NULL;
+#else
+		tmp = 0x00;
+#endif
```

Comment:
Not sure why exactly this should be needed (since the cast should be fine)... But there's absolutely no reason to be using `NULL` here in the first place anyway.
```suggestion
        tmp = 0x00;
```